### ISSUE TITLE
How to set the values of cls and box according to my needs at the beginning of training

### ISSUE THREAD

**Issue by blood7ao** [ultralytics/ultralytics#19137]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

My application requires more accuracy in classification and less in box. How do I set the values of box and cls at the beginning of training, I see that the default value allocation on the documentation is 7.5 and 0.5, I don't understand if there is a proportionality between them or what is the range of their values?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @blood7ao, thank you for your interest in Ultralytics 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com/) for detailed usage information. Many topics, including training tips and hyperparameter adjustments, are well-documented. For specific details about `box` and `cls` weight adjustments, the [Model Training Tips](https://docs.ultralytics.com/guides/model-training-tips/) section might be useful.

If this is a ❓ Question regarding training customization, please provide additional context such as your dataset details, any scripts you've modified, and relevant training logs. This information will help us better understand your needs.

If you've encountered a 🐛 Bug, we would need a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to investigate and debug efficiently.

To ensure compatibility, please upgrade to the latest `ultralytics` package, as improvements and fixes in recent updates might resolve your query. Here's how to do it:

```bash
pip install -U ultralytics
```

YOLO supports seamless training and inference in any of the following verified environments. We recommend trying any of these setups if you are facing reproducibility issues:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud**: See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon**: See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

For real-time collaboration or discussions, feel free to join the Ultralytics community:
- [Discord](https://discord.com/invite/ultralytics) 🎧 for live chats
- [Discourse](https://community.ultralytics.com/) 💬 for detailed discussions
- [Subreddit](https://reddit.com/r/Ultralytics) 👾 to share tips and ask questions

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing, confirming that the repository is stable and operational.

This is an automated response to assist you promptly, but one of our Ultralytics engineers will review your question and provide further insights soon. 😊

**Comment by blood7ao** [comment]
By the way, which pre-training model to choose for 10000 images last and does bigger pre-training model mean better performance

**Comment by glenn-jocher** [comment]
@blood7ao to adjust classification (`cls`) and box (`box`) loss weights in your YOLO11 training, modify these values directly in your training command or config. For your case (higher classification focus), **increase `cls`** (default=0.5, range=0.2-4.0) and **decrease `box`** (default=7.5, range=0.02-0.2). Example: `box=2.0 cls=3.0`. These weights are multipliers in the loss function. 

For details, see the [Hyperparameter Tuning Guide](https://docs.ultralytics.com/guides/hyperparameter-tuning/) and [Train Settings](https://docs.ultralytics.com/modes/train/#train-settings).

**Comment by blood7ao** [comment]
why the box default range = 0.02-0.02,but the default = 7.5,its means 7.5±[0.02-0.2]?

**Comment by glenn-jocher** [comment]
The box range 0.02-0.2 applies specifically to hyperparameter tuning searches (like with Ray Tune), while the default value of 7.5 is the standard loss weight used in normal training. These are separate configurations - the tuning range helps explore effective relative weights during optimization, while the default 7.5 is the fixed weight for standard training. You can adjust them directly in training via `box=7.5 cls=0.5` without any conversion formula. For details, see the [Hyperparameter Tuning Guide](https://docs.ultralytics.com/guides/hyperparameter-tuning/).
